src/app/activity/bounty/Publish.ts

```diff
@@ -29,8 +29,8 @@ export const publishBounty = async (publishRequest: PublishRequest): Promise<any
 	const bountyChannel: TextChannel = dbBountyResult.createdInChannel ? await DiscordUtils.getTextChannelfromChannelId(dbBountyResult.createdInChannel) :
 		 await guildMember.client.channels.fetch(dbCustomerResult.bountyChannel) as TextChannel;
 	const bountyMessage: Message = await BountyUtils.canonicalCard(dbBountyResult._id, publishRequest.activity, bountyChannel);
-    await guildMember.send({ content: `Bounty published to ${bountyChannel.name} <${bountyMessage.url}> and the website! <${process.env.BOUNTY_BOARD_URL}${bountyId}>` });
-	Log.info(`bounty published to ${bountyChannel.name}`);
+    await guildMember.send({ content: `Bounty published to \`${(bountyMessage.channel as any).name || bountyChannel.name}\` <${bountyMessage.url}> and the website! <${process.env.BOUNTY_BOARD_URL}${bountyId}>` });
+	Log.info(`bounty published to ${(bountyMessage.channel as any).name || bountyChannel.name}`);
 
 	// Remove old publish preview
 	if (dbBountyResult.creatorMessage !== undefined) {
```

Comment:
In the below code, probably don't need to error out. If the person deleted their draft bounty card, then used "/bounty publish" to publish it using the ID, that should still work OK.

Reply:
when do we need this logic? I see that we are already deleting the preview on https://github.com/thedarkrai07/bountybot-/blob/64877183669b4da430817ea8013c7dcca99d6f00/src/app/utils/BountyUtils.ts#L408

Reply:
Before we had the canonical card method, we were storing the message pointers in creatorMessage and claimantMessage. If there were old bounties in a Draft state at the time the new canonical card code went in, we needed to handle those. This code will eventually be obsolete.